### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by boda4ek**

### CODE REVIEW

File: src/getCoinCombination.test.js [hunk 1]
```diff
@@ -7,4 +7,67 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('cents = 0', () => {
```
Comment (murat-yesimbietov) [inline src/getCoinCombination.test.js:11]: the summary of the test is not clear, rewatch the videos about naming tests:
https://mate.academy/en/learn/javascript-testing/tdd-basics-js?section=video&videoId=1764
https://mate.academy/en/learn/javascript-testing/tdd-basics-js?section=video&videoId=1765

File: src/getCoinCombination.test.js [hunk 2]
```diff
@@ -7,4 +7,67 @@ describe('getCoinCombination', () => {
...
+  });
+
+  it('0 < cents < 5', () => {
+    const actual = getCoinCombination(3);
+    const expected = [3, 0, 0, 0];
+
+    expect(actual).toEqual(expected);
+  });
+
+  it('cents = 5', () => {
```
Comment (murat-yesimbietov) [inline src/getCoinCombination.test.js:25]: Recall test design techniques and apply these techniques (Equivalence class and boundary value at least)

**Review by murat-yesimbietov — CHANGES_REQUESTED** [review]

File: src/getCoinCombination.test.js [hunk 3]
```diff
@@ -8,63 +8,112 @@ describe('getCoinCombination', () => {
       .toBeInstanceOf(Function);
   });
 
-  it('cents = 0', () => {
+  it(`should returt [0, 0, 0, 0] if 'cents' = 0`, () => {
```
Comment (murat-yesimbietov) [inline src/getCoinCombination.test.js:11]: little typo: returt

File: src/getCoinCombination.test.js [hunk 4]
```diff
@@ -8,63 +8,112 @@ describe('getCoinCombination', () => {
...
 
-  it('0 < cents < 5', () => {
+  it(`should returt [1, 0, 0, 0] if 'cents' = 1`, () => {
+    const actual = getCoinCombination(1);
+    const expected = [1, 0, 0, 0];
+
+    expect(actual).toEqual(expected);
+  });
+
+  it(`should returt [3, 0, 0, 0] if 0 < 'cents' < 5`, () => {
```
Comment (murat-yesimbietov) [inline src/getCoinCombination.test.js:25]: If `cents = 2`, should the function return `[3, 0, 0, 0]`?

File: src/getCoinCombination.test.js [hunk 5]
```diff
@@ -8,63 +8,112 @@ describe('getCoinCombination', () => {
...
+
+  it(`should returt [2, 1, 1, 2] if 'cents' = 67`, () => {
     const actual = getCoinCombination(67);
     const expected = [2, 1, 1, 2];
 
     expect(actual).toEqual(expected);
   });
 
-  it('cents is NaN', () => {
+  it(`should returt [0, 0, 0, 0] if'cents' is NaN`, () => {
```
Comment (murat-yesimbietov) [inline src/getCoinCombination.test.js:116]: but the expected value is `[NaN, NaN, NaN, NaN]`

**Review by murat-yesimbietov — CHANGES_REQUESTED** [review]
Little changes are required

**Review by murat-yesimbietov — APPROVED** [review]
